includes/admin/class-wc-stripe-settings-controller.php

```
@@ -216,7 +216,14 @@ public function admin_scripts( $hook_suffix ) {
 			// Show the Stripe Tax banner only if OC is enabled
 			&& $is_oc_enabled;
 
-		$is_ap_available_for_account = WC_Stripe_Helper::is_adaptive_pricing_available_for_account();
+		$is_checkout_sessions_available       = false;
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $is_checkout_sessions_available. Keep variable name length under 20. (undefined)

(LongVariable)